ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDPayment.java

```diff
@@ -11,7 +11,6 @@
     private String id;
     private BigDecimal amount;
     private String reference;
-    private String description;
```

Comment:
Removing description field from ccd adapter does not look correct?

Reply:
It's fine as we are not using it.

Reply:
All actioned!